vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataType.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.contrib.vlm.gdal
+
+import com.azavea.gdal.GDALWarp
+
+
+object GDALDataType {
```

Comment:
Mostly for @echeipesh:

I think we can merge everything into contrib now, and after that to port some things into geotrellis-gdal that should be necessarily there. What to you think? @echeipesh @jamesmcclain

Reply:
I think it needs to be tested on staging before we do any merging.